codecov.yml

```diff
@@ -6,23 +6,16 @@ codecov:
         require_ci_to_pass: no
 
 coverage:
-    precision: 2
-    round: down
-    range: '70...100'
-
     status:
         project:
             default:
-                threshold: 1
                 informational: true
             codewhisperer:
-                target: 70%
                 paths:
                     - packages/core/src/codewhisperer/*
                 flags:
                     - 'codewhisperer'
             amazonqFeatureDev:
-                target: 70%
```

Comment:
rather than removing this, can we try adding `target: 70%` to stepFunctions etc. those are the ones that are failing